newsrevenuehub/rev-engine
# DEV-2534: ignore test contributions in live mode and vice versa

Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)

N/A

#### What's this PR do?

Ignore test events in live mode and live events in test mode

#### Why are we doing this? How does it help us?

The current webhook generates too much noise as live events cannot be processed in test (and vice versa), resulting in a lot of noise in Sentry

#### How should this be manually tested?

- Deploy to production
- Submit a contribution in [TEST](https://hub.revengine-test.org/donate/)
- Got to [logs](https://my.papertrailapp.com/systems/rev-engine-prod/events) and assert that you can see logs for `live mode event %s for account %s received while in test mode; ignoring`

#### Did this PR make changes to the user interface that may require changes to the user-facing docs?

N/A

#### Have automated unit tests been added? If not, why?

N/A

#### How should this change be communicated to end users?

N/A

#### Are there any smells or added technical debt to note?

N/A

#### Has this been documented? If so, where?

N/A

#### What are the relevant tickets? Add a link to any relevant ones.

[DEV-2534]

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?

N/A

#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:

N/A

[DEV-2534]: https://news-revenue-hub.atlassian.net/browse/DEV-2534?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
